files/image_config/copp/copp_cfg.j2

```diff
@@ -79,12 +79,10 @@
 		    "trap_ids": "lldp",
 		    "trap_group": "queue4_group3"
 	    },
-{% if not (DEVICE_METADATA is defined and DEVICE_METADATA['localhost'] is defined and DEVICE_METADATA['localhost']['type'] is defined and DEVICE_METADATA['localhost']['type'] != "ToRRouter") %}
-	    "dhcp": {
+	    "dhcp_relay": {
```

Comment:
There is a requirement to not trap those DHCP packet on non-ToR deployment. This is a change is requirement. I'd suggest to have copp_mgr adapt and support his requirement.

Reply:
This will be handled in hostcfgd where dhcp_relay feature will be disabled on non ToR deployment

Reply:
CPU trap will be on T1/T2 deployment, right? It is now up to ctrl plane to forward those packet. This is a change in behavior.

Reply:
CPU trap will be dynamically enabled based on feature. If a feature is disabled, then the corresponding trap will not be installed. COPP manager handles this based on the fact that feature table and copp table have same keys. That's the reason we changed dhcp keyword in copp file to dhcp_relay to match the feature, thereby making it to listen to feature notifications and installing trap only if enabled

Reply:
Can you please point me to this code? I was not aware of that.

Reply:
https://github.com/Azure/sonic-swss/blob/a3bc89bf3cff4bdf2a6ba80a0f4dd9fedb8f65ba/cfgmgr/coppmgr.cpp#L56

Reply:
Thanks @dgsudharsan!